sascha_willems/README.md

```diff
@@ -0,0 +1,16 @@
+# Vulkan Samples Assets
+
+Licenses and attributions for the various scenes and textures used in this sub-folder.
+
+## Scenes
+
+- Buggy.gltf taken from the [Khronos gltf Sample Models repository](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Buggy).
```

Comment:
should be removed now, since it doesn't exist anymore.

Reply:
Has been removed.